README.md

```diff
@@ -99,6 +99,26 @@ point. You can, however, change this behaviour by invoking such
 commands with a prefix argument. For instance: `C-u C-c C-v` will ask
 for the symbol you want to show the docstring for.
 
+#### *WARNING* - Standard REPLs
+
+Note that if you decide _NOT_ to use the socket repl, it is highly recommended
+you disable output coloring and/or readline facilities: `inf-clojure` does not
+filter out ASCII escape characters at the moment and will not behave correctly.
```

Comment:
Why filter them out? Why not simply apply them the same way we do in CIDER? It's quite trivial.